drafts/use-cases/1.entry-point.md

```diff
@@ -0,0 +1,108 @@
+## Entry point
+
+### Story
+As an application user
+I want my application to consume an entry point for the Web API by providing a base URL
+So I can start managing my events.
+
+### Usage
+```javascript
+var client = new HydraClient();
+var data = client.get("/api");
```

Comment:
Since this entry point discovery is something that needs to be performed often unless cached, should there be any mention of caching here? It surely applies to all URIs consumed by the Hydra Client, but the entry point is a contention point.

Also, if we mention caching, in what detail should we go with regards to cache lifetime, invalidation, etc.? Just refer to RFC 7234 or what?